src/config.ts

```diff
@@ -76,17 +76,14 @@ export function parseConfig(opts: Partial<Config>): Config {
       'i18next',
       'i18n',
     ]),
-    tFunctionNames: coalesce(opts.tFunctionNames, ['t']),
+    customTFunctionNames: coalesce(opts.customTFunctionNames, ['t']),
```

Comment:
Note for v1: the default value here should be an empty array (to be consistent with other `custom*` options). However, we may want to allow user to disable default `t` function extraction (not sure if it's really useful).